source/extensions/filters/common/ext_authz/ext_authz_http_impl.h

```diff
@@ -41,6 +41,7 @@ class RawHttpClientImpl : public Client,
   void onFailure(Http::AsyncClient::FailureReason reason) override;
 
 private:
+  ResponsePtr messageToResponse(Http::MessagePtr& message);
```

Comment:
Can we avoid passing a reference to a unique_ptr? Can this just be `const Http::Message&`?

Reply:
There few `non-const` member functions in Message that need to be called. 
Is it ok if I change it to : 

```ResponsePtr messageToResponse(Http::MessagePtr message)``` 

Then just move it? 

```messageToResponse(std::move(message))```